lib/LicenseTermsList/LicenseTermsList.js

```diff
@@ -19,9 +19,18 @@ export default class LicenseTermsList extends React.Component {
       label: PropTypes.string,
       name: PropTypes.string,
       primary: PropTypes.bool,
+      defaultInternal: PropTypes.bool,
     })),
   };
 
+  renderVisibility = (internal, term) => {
+    if ((internal === false) || (!internal && term.defaultInternal === false)) {
+      return <span data-test-term-visibility={term.name}>{<FormattedMessage id="stripes-erm-components.term.internalFalse" />}</span>;
```

Comment:
Let's break the two spans in this method into multiple lines. Also, it doesn't look like we need the braces around the `FormattedMessage`s